packages/file-format-ts/scripts/generate.ts

```diff
@@ -18,8 +14,8 @@ type SchemaMap = {
   [key: string]: JSONSchema7
 }
 
-const generate = (version: string, schemas: any) => {
-  const outFile = `${version}-types.ts`
+const generate = (schemas: any) => {
```

Comment:
At least what I read from the documentation `@sketch-hq/sketch-file-format` should be typed (or maybe provided by the [`@sketch-hq/sketch-file-format-ts`](https://www.npmjs.com/package/@sketch-hq/sketch-file-format-ts)

meaning that there should be a type for the `schemas` here instead of the `any`. 
However, I'm not sure which type we should use, maybe I can ping @jedrichards as he contributed on both of those projects. 

If there is such type, then I believe it would be far better to use it here.

Reply:
It's the type `Schemas`https://github.com/sketch-hq/sketch-document/blob/main/packages/file-format/src/index.ts#L13

you need to add `export type Schemas = {` so the type can be imported from the other file.

Then, import in `generate.ts`
```
import sketchSchemas, { Schemas } from '@sketch-hq/sketch-file-format'
```

and type better the input
```
const generate = (schemas: Schemas) => {
```

That would make the PR better typed.

Reply:
Good point, though `schemas` is of type `Schemas` which is not exported.

https://github.com/sketch-hq/sketch-document/blob/13d3d3275623111ac72ed9a4e1768244512182fc/packages/file-format/src/index.ts#L13-L33

Maybe something that should be exported, too. Or would the type be declared somewhere automatically so it could be used here?

Reply:
You can export it as I wrote in the comment above.

`export type Schemas { `

and the import it as

`import sketchSchemas, { Schemas } from '@sketch-hq/sketch-file-format'`

Reply:
@ivanmalagon you clearly typed quicker than me 😅